JS401_Simple_Dynamic_with_nodeJS_challenges/create_basic_template_engine_in_node_challenge/utilities.js

```diff
@@ -0,0 +1,9 @@
+function merge(content, values) {
+    for (let key in values) {
```

Comment:
use `const` instead of `let`
```suggestion
    for (const key in values) {
```